packages/wallet-adapter-core/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @aptos-labs/wallet-adapter-core
 
+## 3.1.0
+
+### Minor Changes
+
+- cce3d13: Upgraded @aptos-labs/ts-sdk to version 1.0.0
```

Comment:
this is... not how it should be. have you run `pnpm run changeset` from the root?

Reply:
I did run it from root